src/components/views/TrackView.js

```diff
@@ -8,7 +8,7 @@ import RiderSearchContainer from '../../containers/RiderSearchContainer'
 class TrackView extends Component {
     render() {
       const {judgments} = this.props;
-        const {search} = this.props;
+      const {search} = this.props;
       const {raceGroups} = this.props;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "raceGroups" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'raceGroups' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AUnusedVariable)